static/js/transform-facets.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * A Facets component transformFacets function which applies the configuration
+ * specified by the Facets 'fields' option
+ * 
+ * @param {DisplayableFacet[]} facets from answers-core
+ * @param {FilterOptionsConfig} config from answers-search-ui
+ * @returns {DisplayableFacet[]} from answers-core
+ */
+export default function transformFacets (facets, config) {
+  if(!config || !('fields' in config)) {
+    return facets;
+  }
+
+  return facets.map(facet => {
+    const isFacetFieldConfigSpecified = facet.fieldId in config.fields;
+    if (!isFacetFieldConfigSpecified) {
+      return facet;
+    }
+    const fieldConfig = config.fields[facet.fieldId];
```

Comment:
Would it be more accurate to call this `facetConfig`? That name may make more sense in the context of this method. We are looking for custom facet configuration specified by the HHs and melding that with the `DisplayableFacet`s.

Reply:
I was thinking fieldConfig because it's keyed by `fieldId`, but I think `facetConfig` does make more sense